debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
Yes the spec file would see duplication in this case. We could use rpm tools on Ubuntu. However they are not native to a debian system. Fur example if you asked OBS to build Debian package it can't do this additional step to make an room and then convert to debian. This will allow anybody to build using a OBS like build infra which expects deb packaging to be advisor or if the box. Now, we should definitely try to remove  duplication wherever possible. Is there a way to refactor common stuff into common files?